content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -29,7 +29,7 @@ To work with Kubernetes objects--whether to create, modify, or delete them--you'
 Every Kubernetes object includes two nested object fields that govern the object's configuration: the object *spec* and the object *status*. The *spec*, which you must provide, describes your *desired state* for the object--the characteristics that you want the object to have. The *status* describes the *actual state* of the object, and is supplied and updated by the Kubernetes system. At any given time, the Kubernetes Control Plane actively manages an object's actual state to match the desired state you supplied.
 
 
-For example, a Kubernetes Deployment is an object that can represent an application running on your cluster. When you create the Deployment, you might set the Deployment spec to specify that you want three replicas of the application to be running. The Kubernetes system reads the Deployment spec and starts three instances of your desired application--updating the status to match your spec. If any of those instances should fail (a status change), the Kubernetes system responds to the difference between spec and status by making a correction--in this case, starting a replacement instance.
+For example, Kubernetes Deployment is a controller build upon the basic objects. It can represent an application running on your cluster. When you create the Deployment, you might set the Deployment spec to specify that you want three replicas of the application to be running. The Kubernetes system reads the Deployment spec and starts three instances of your desired application--updating the status to match your spec. If any of those instances should fail (a status change), the Kubernetes system responds to the difference between spec and status by making a correction--in this case, starting a replacement instance.
```

Comment:
Hello @ptux . Since this page is about `working-with-objects`, I think it is okay to describe `Deployment` as a Kubernetes object. The paragraph further elaborates on the `Deployment` manifest and configuration.
See, https://kubernetes.io/docs/concepts/workloads/controllers/deployment/, which clarifies Deployment controllers and Deployment objects,
`You describe a desired state in a Deployment object, and the Deployment controller changes the actual state to the desired state at a controlled rate.`

Reply:
I got it.